model_zoo/mnist/mnist_pytorch_custom_dataset.py

```diff
@@ -0,0 +1,180 @@
+# Copyright 2020 The ElasticDL Authors. All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+We can use the following command to submit a training job with the script.
+elasticdl train \
+  --image_name=elasticdl:dev_allreduce  \
+  --training_data=/data/mnist/train/images.csv \
+  --job_command="python -m model_zoo.mnist.mnist_pytorch_custom_dataset" \
+  --num_epochs=1 \
+  --num_minibatches_per_task=2 \
+  --minibatch_size=64 \
+  --num_workers=2 \
+  --worker_pod_priority=0.5 \
+  --master_resource_request="cpu=0.2,memory=1024Mi" \
+  --master_resource_limit="cpu=1,memory=2048Mi" \
+  --worker_resource_request="cpu=0.3,memory=1024Mi" \
+  --worker_resource_limit="cpu=1,memory=2048Mi" \
+  --envs="USE_TORCH=true" \
+  --job_name=mnist-allreduce \
+  --image_pull_policy=Never \
+  --distribution_strategy=AllreduceStrategy \
+  --need_elasticdl_job_service=false \
+"""
+
+import argparse
+import linecache
+import os
+
+import cv2
+import numpy as np
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.optim as optim
+from torch.utils.data import DataLoader, IterableDataset
+
+from elasticai_api.pytorch.optimizer import DistributedOptimizer
+from elasticdl.python.allreduce.pytorch_controller import (
```

Comment:
Update the package from `elasticdl.python.allreduce.pytorch_controller ` to `elasticai_api.pytorch.controller `

Reply:
Done!